packages/renderer/src/lib/table/Table.svelte

```diff
@@ -66,6 +67,14 @@ function sort(column: Column<any>) {
   data = data.sort(comparator);
 }
 
+onMount(async () => {
+  const column: Column<any> | undefined = columns.find(column => column.title === defaultSortColumn);
+  if (column) {
+    sortCol = column;
+    sortAscending = true;
+  }
```

Comment:
```suggestion
    sort(column);
  }
```

Reply:
it is not sorting default column after opening the page.

https://github.com/containers/podman-desktop/assets/620330/e2542138-f0ce-4da8-8627-e16954d80624

Reply:
The intention (for now) wasn't to provide a way to sort the initial data by any column, but just to set the initial sort indicator _when the initial data is already sorted_.

The suggested change doesn't work because most of our existing pages are dynamic / lazy-loading - data is empty when the page is shown (so onMount would sort nothing) and then data is set. To be able to set an arbitrary sort column we'd need to sort after the data is received.

We may need to support this at some point, but since our current pages don't need it I would prefer to release what's here and defer arbitrary default sorting until later.